pwnlib/tubes/tube.py

```diff
@@ -99,7 +137,7 @@ def recv(self, numb = None, timeout = default):
             True
             >>> with context.local(log_level='debug'):
             ...    _ = t.recv() # doctest: +ELLIPSIS
-            [...] Received 0xc bytes:
+            [DEBUG] Received 0xc bytes:
```

Comment:
Can probably break if NOTERM is not set; not sure tho

Reply:
It's with `DEBUG` here: https://github.com/Gallopsled/pwntools/blob/f2f55f3e1c26e41e7b23f2fab0ff41baf7ebb27c/pwnlib/tubes/tube.py#L1066

Reply:
Sure but what if it is '[\33[91mDEBUG\33[m] Something'`? I think this was the reason.

Reply:
I replaced both with `[...]`.